### PR TITLE
chore: mention more token+cookie prefixes

### PR DESCRIPTION
**Pull request by alnr**

### CODE REVIEW

**Review by jonas-jonas — APPROVED** [review]

File: docs/security-compliance/token-formats.mdx [hunk 1]
```diff
@@ -25,13 +25,19 @@ security breaches and unauthorized access.
...
 Developers can configure the expiration time and cookie options for Ory session cookies to fit their application's specific needs.
 
-In summary, Ory session cookies are prefixed with ory*session* and are essential for maintaining user sessions. They can be
-customized to fit specific application needs and are secure and tamper-proof.
+## Other Ory cookies
+
+Ory Network will also issue other cookies to protect against CSRF attacks, provide load balancing, and other security features.
+These cookies will be prefixed as follows:
+
+- `ory_`: Ory-internal cookie (CSRF protection etc)
```
Comment (vinckr) [inline docs/security-compliance/token-formats.mdx:38]:
```suggestion
- `ory_`: Ory-internal cookie (CSRF protection)
```

IMO either list explicitly or not (I assume its not that much)

**Review by jonas-jonas — APPROVED** [review]
would be cool if we could dynamically create the full cookie name from the project slug, but that will require some changes around getting the current project into docs.

**Review by vinckr — APPROVED** [review]
lgtm, small nit

File: docs/security-compliance/token-formats.mdx (hunk 1 above)
Comment (alnr) [reply docs/security-compliance/token-formats.mdx:38]: We have a bunch of other cookies which I'd only list of someone really, really, really needs to know their names for some inexplicable reason. GDPR certainly doesn't require listing them all by name.

**Pull request merged**